src/pages/en/guides/content-collections.mdx

```diff
@@ -187,131 +127,145 @@ defineCollection({
 const allBlogPosts = await getCollection('blog');
+```
+
+`getCollection` also takes a "filter" callback as an optional second argument. This allows you to filter your query based on an entry's `id`, `slug`, or `data` (frontmatter) properties.
 
-// Only return posts with `draft: true` in the frontmatter
+```js
+// Example: Only return entries with `draft: true` frontmatter
+import { getCollection } from 'astro:content';
 const draftBlogPosts = await getCollection('blog', ({ data }) => {
   return data.draft === true;
 });
----
 ```
 
-#### Querying nested directories
+#### Filtering nested directories
 
-The filter function can also be used to query for nested directories within a collection. Since the `id` includes the full nested path, you can filter by the start of each `id` to only return items from a specific nested directory:
+The filter argument lets you query for nested directories within a collection. Since the `id` includes the full nested path, you can filter by the start of each `id` to only return items from a specific nested directory:
 
-```astro
----
+```js
+// Example: Only return entries from `src/content/docs/en/`
 import { getCollection } from 'astro:content';
 const enDocs = await getCollection('docs', ({ id }) => {
-  // Return all entries in `src/content/docs/en/`
   return id.startsWith('en/');
 });
----
 ```
 
-### `getEntry()`
+### `getEntryBySlug()`
 
-`getEntry()` is function that returns a specific entry in a collection by entry ID (file path relative to the collection). Both of these are required parameters.
+`getEntryBySlug()` is function that returns a specific entry in a collection. It takes two arguments: the collection name and the content entry's `slug` (URL-ready).
 
-```astro
----
-import { getEntry } from 'astro:content';
-
-const enterprise = await getEntry('blog', 'enterprise.md');
----
+```js
+import { getEntryBySlug } from 'astro:content';
+const entry = await getEntryBySlug('blog', 'enterprise');
 ```
 
-### Data returned from a collection query
+### Using content in templates
 
-`getCollection()` and `getEntry()` will return entries that include:
- - `id` - a unique ID using the file path relative to `src/content/[collection]`.
- - `slug` - a URL-ready slug. Defaults to the `id` parsed by [github-slugger](https://github.com/Flet/github-slugger).
- - `data` - an object of frontmatter properties inferred from your collection schema. Defaults to `any` if no schema is configured.
- - `body` - a string containing the raw, uncompiled body of the Markdown or MDX document.
- - `render()` - a function that returns the compiled body of the Markdown or MDX document via the `<Content />` component ([See complete documentation](#rendering-entry-content)).
+Once you have queried your content, you can use the returned entries inside of your Astro pages and components. For example, you can include frontmatter in a component template, or pass an entire content entry object as a prop.
```

Comment:
```suggestion
Once you have queried your content, you can use the returned entries inside of your Astro pages and components. For example, you can include frontmatter in a component template from a `data` object, or pass an entire content entry object as a prop.

See the [entire list of properties returned](/en/reference/api-reference/#getcollection) from queries.
```

Since we've lost the list of returned data properties on this page, I'd argue for being just a tiny bit more explicit somehow about what is returned and available for rendering here if we want to offload the bulk of the info to another page. I think it is appropriate here to at least say that you get a `data` object of your frontmatter properties, and `slug`, for example. 

And, somewhere (I'm just making all these notes here, but I really mean incorporated around this section), I would explicitly link back to the properties on `api-reference` (assuming that's where they'll go).

Reply:
done!